QGL/drivers/APS2Pattern.py

```diff
@@ -770,7 +770,7 @@ def start_new_seq():
 					seqs['mod_phase'][-1] = np.append(seqs['mod_phase'][-1], freq[nco_select]*np.arange(count) + accumulated_phase[nco_select] + phase[nco_select] + frame[nco_select])
 					accumulated_phase += count*freq
 				else:
-					phase_rad = 2*np.pi * (instr.payload & 0xffffffff) / 2**28
+					phase_rad = 2*np.pi * np.int32(instr.payload & 0xffffffff) / 2**28
```

Comment:
This overflows: 
> Python int too large to convert to C long

Does ` np.int32((instr.payload & 0xffffffff) / 2**28)` achieve the same goal?